### DISCUSSION TITLE
Mosaic batch process for an animation but it can't find the mask

### DISCUSSION THREAD

**Discussion by realdreamachine** [lllyasviel/stable-diffusion-webui-forge#2023] [General]
Hello can anybody here help please? I am using the Mosaic extension. Despite giving the right local address/folder for the mask PNG image, it won't recognise it. Promising results with current settings turning a 704 × 576 movie into widescreen 1280 x 576 call for a batch process and I'd like to do it now.

It says "Inpaint batch mask directory (required for inpaint batch processing only)" But when I put it in I get..

NotADirectoryError: [WinError 267] The directory name is invalid: 'C:/Users/Owner/AppData/Local/Temp/gradio/f967c89bdddc4c2cbc66c2010878fb7a83350b45/mask.png'

What am I missing or doing wrong there?

**Comment by DenOfEquity** [comment]
You're giving the full filename, not the directory. Using the temporary Gradio file is potentially also unreliable, better to save the mask somewhere. I don't think the Mosaic extension supports batch processing, so you'd have to produce the expanded mosaic images one at a time anyway.
